app/api/constituency_lookup/[postcode]/[[...address]]/route.ts

```diff
@@ -18,27 +18,25 @@ console.log(`***** dbPath is [${dbPath}] *****`);
 // TODO: Next.js doesn't cache POSTs - potentially change to a GET of format:
 // constituency_lookup/{postcode}/{addressSlug}
 // which should enable caching
-export async function POST(request: NextRequest) {
+export async function GET(request: NextRequest, context) {
```

Comment:
Different topic on this line - the Vercel deploy is failing with:
`Type error: Parameter 'context' implicitly has an 'any' type.`

I probably need to add a typescript check to the linter/pretty Github action check.